frontend/components/doctor/patient-charts-overview.tsx

```diff
@@ -66,29 +83,31 @@ export default function PatientChartsOverview({ patientList }: { patientList: Pa
                             <ScatterChart
                                 statuses={extractStatuses(patientList)}
                                 transformDataFn={transformWeightTempData}
-                                day={DAY.TODAY}
+                                day={scatterChartDate}
                                 title={"Patients' Temperature and Weight Today"}
-                                h={260}
-                                w={400}
-                            />
-                        </TabPanel>
-                        <TabPanel>
-                            <ScatterChart
-                                statuses={extractStatuses(patientList)}
-                                transformDataFn={transformWeightTempData}
-                                day={DAY.YESTERDAY}
-                                title={"Patients' Temperature and Weight Yesterday"}
-                                h={260}
+                                h={275}
                                 w={400}
                             />
+                            <Center>
+                                <label htmlFor="WeightTempDate">Date picked:&emsp;</label>
+                                <input
+                                    id="scatterChartInput"
+                                    type="date"
+                                    name="WeightTempDate"
+                                    max={today}
+                                    onChange={e => pickDate(e.target.value, "scatter")}
+                                    defaultValue={today}
+                                    style={datePickerStyle}
+                                />
+                            </Center>
                         </TabPanel>
                         <TabPanel>
                             <ScatterChart
                                 statuses={extractStatuses(patientList)}
                                 transformDataFn={transformWeightTempData}
                                 day={DAY.ALL}
                                 title={"Patients' Temperature and Weight of All Time"}
-                                h={260}
+                                h={275}
                                 w={400}
```

Comment:
do these set widths and heights look good on mobile?

Reply:
Pretty bueno if I do say so myself 🥸